components/form-pod/tests/unit/form-pod-panel-footer.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react';
+import { GEL } from '@westpac/core';
+import wbc from '@westpac/wbc';
+import { PanelFooter } from '@westpac/form-pod';
+
+import { overridesTest } from '../../../../helpers/tests/overrides-test';
+import { nestingTest } from '../../../../helpers/tests/nesting-test';
+import { FormPodPanelFooterProps } from '../../src/FormPodPanelFooter';
+
+overridesTest({
+	name: 'form-pod',
+	overrides: ['Footer'],
+	Component: (props: any) => {
+		const LeftItem = <div>Left Item</div>;
+		const RightItem = <div>Right Item</div>;
+		return <PanelFooter left={LeftItem} right={RightItem} {...props} />;
+	},
+});
```

Comment:
`overrides: ['Footer'],`
I was wondering if this could or should include 'FooterItem'

Reply:
When adding 'Footer', it failed on the FooterItem overrides, but when rendering just one FooterItem, by removing either left or right prop, the test passed

Reply:
hi @wove5 , currently I could not find an alternative way to test `FooterItem`, without updating the `overridesTest` function due to the reason it's currently giving a same `testid` to both of those elements.
Can we skip `FooterItem` for now and add it to the TODO list once `overridesTest` function gets improved?

Reply:
just left a TODO comment on it :)
will handle this matter in different PR shortly